src/modules/TimeScale.js

```diff
@@ -632,31 +642,17 @@ class TimeScale {
     let yrCounter = 0
     let unit = 'minute'
 
-    let remainingSecs = 60 - firstVal.minSecond
-
-    let firstTickPosition = remainingSecs * secondsWidthOnXAxis
-    let firstTickValue = firstVal.minMinute + 1
-    let minute = firstTickValue + 1
+    let remainingSecs = 60 - currentSecond
+    let firstTickPosition =
+      (remainingSecs - currentMillisecond / 1000) * secondsWidthOnXAxis
```

Comment:
On some zoom levels label is positioned 3+ pixels off comparing to tooltip line. Accounted for milliseconds to fix that.